goja.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/dop251/goja"
+	"github.com/dop251/goja_nodejs/console"
+	"github.com/dop251/goja_nodejs/require"
+)
+
+func newVMWithRequire() (*goja.Runtime, *require.RequireModule) {
+	registry := new(require.Registry)
+	vm := goja.New()
+	req := registry.Enable(vm)
+	console.Enable(vm)
+
+	vm.SetFieldNameMapper(goja.UncapFieldNameMapper())
+	// https://github.com/dop251/goja#mapping-struct-field-and-method-names
+	// use this if we need optionally uncapitalises
+	// vm.SetFieldNameMapper(goja.TagFieldNameMapper("json", true))
```

Comment:
this is for mapping fields or methods names between go-like and js-like

and we can tag if we want optionally uncapitalise them